README.md

```diff
@@ -47,6 +47,9 @@ as they may have been used for past encryption operations.
   * * OCI KMS (uses envelopes)
   * * Tencent Cloud KMS (uses envelopes)
   * * Vault Transit mount
+  * Supports generic integrations
+  * * PKCS11
+  * * KMIP
```

Comment:
@cipherboy I added KMIP and also PKCS11 (if you don't mind) which was missing in the root README

Reply:
Ah, perfect. Thanks!